<?xml version="1.0"?>
<launch>

    <arg name="loop_rate" default="20.0" />
    <arg name="distance_between_wheels" default="0.32" /> <!-- CORRECT VALUE HERE -->
    <arg name="wheel_diameter" default="0.066" />

    <!-- odometry -->
    <node pkg="rdl_turtlebot3_navigation" type="odometry.py" name="turtlebot3_odometry" output="screen">
        <param name="loop_rate" value="$(arg loop_rate)"/>
        <param name="distance_between_wheels" value="$(arg distance_between_wheels)"/>
        <param name="wheel_diameter"  value="$(arg wheel_diameter)"/>
        <remap from="/joint_states" to="/turtlebot3_burger/joint_states"/>
    </node>

</launch>
